apps/openmw/mwgui/tooltips.cpp

```diff
@@ -17,6 +17,7 @@
 #include "../mwworld/class.hpp"
 #include "../mwworld/esmstore.hpp"
 #include "../mwmechanics/spellcasting.hpp"
+#include "../mwmechanics/mechanicsmanagerimp.hpp"
```

Comment:
You should include `../mwbase/mechanicsmanager.hpp` instead (the interface, not the implementation).